src/client/gui/lib/tray_menu.dart

```diff
@@ -229,13 +180,35 @@ Future<void> _updateTrayMenu(
         'start',
         label: 'Start',
         enabled: startEnabled,
-        callback: startCallback(name),
+        callback: (_, __) => grpcClient.start([name]),
       );
       await submenu.addLabel(
         'stop',
         label: 'Stop',
         enabled: stopEnabled,
-        callback: stopCallback(name),
+        callback: (_, __) => grpcClient.stop([name]),
+      );
+      await submenu.addSeparator('separator');
+      await submenu.addLabel(
+        'open',
+        label: 'Open in Multipass',
+        callback: (_, __) {
```

Comment:
I think this should bring the window up if minimized or hidden behind another window. Currently it only switches to the instance page in the GUI and only toggles the window if untoggled (at least that is what I observe). I think this is something to revisit later. This whole toggle approach doesn't feel quite right to me just yet...

Reply:
Yes, it would be great if it put the window on top of other windows. Unfortunately that functionality is not provided by the window manager... But yeah, I agree that the toggle thing is a bit weird...

Reply:
OK, don't worry for now. Just please keep track of this to circle back around later on.